src/sbp_fileio.c

```diff
@@ -81,25 +83,27 @@ static void read_cb(u16 sender_id, u8 len, u8 msg[], void* context)
   len += cfs_read(f, buf + len, readlen);
```

Comment:
Let's say the original message is 16 bits long as sent (len=16).  Lets say the readlen field of the message is it's max value (256).  the cfs_read operation just overran buf.